### PR TITLE
[Doc] Document the usage of LS_JAVA_OPTS environment variable

### PR DESCRIPTION
**Pull request by andsel**
## Release notes

[rn:skip] 

## What does this PR do?

Update the guide section where it's described how to configure Logstash to document the `LS_JAVA_OPTS` environment variabl eand its usage

## Why is it important/What is the impact to the user?

Let the user to discore documentation about `LS_JAVA_HOME`

## Checklist

- [x] My code follows the style guidelines of this project
- ~~[ ] I have commented my code, particularly in hard-to-understand areas~~
- [x] I have made corresponding changes to the documentation
- ~~[ ] I have made corresponding change to the default configuration files (and/or docker env variables)~~
- ~~[ ] I have added tests that prove my fix is effective or that my feature works~~

## Author's Checklist

- [ ]

## Related issues

-

### CODE REVIEW

File: docs/static/config-details.asciidoc [hunk 1]
```diff
@@ -111,6 +111,18 @@ java -XX:+PrintFlagsFinal -version | grep ThreadStackSize
 Depending on the default stack size, start by multiplying by 4x, then 8x, and
 then 16x until the overflow error resolves.
 
+[[ls-java-opts]]
+==== Using `LS_JAVA_OPTS`
+
+`LS_JAVA_OPTS` environment variable could be used to configure jvm options without editing the `jvm.options` <<settings-files,settings file>>.
```
Comment (robbavey) [inline docs/static/config-details.asciidoc:117]:
```suggestion
The `LS_JAVA_OPTS` environment variable can also be used to override JVM settings in the `jvm.options` file <<settings-files,settings file>>.
```

File: docs/static/config-details.asciidoc [hunk 2]
```diff
@@ -111,6 +111,18 @@ java -XX:+PrintFlagsFinal -version | grep ThreadStackSize
...
+The content of the variable is combined with the one present in the `jvm.options`.
```
Comment (robbavey) [inline docs/static/config-details.asciidoc:118]:
```suggestion
The content of this variable is additive to options configured in the `jvm.options` file, and will override any settings that exist in both places
```

File: docs/static/config-details.asciidoc [hunk 3]
```diff
@@ -1,7 +1,7 @@
 [[jvm-settings]]
 === JVM settings
 
-Configure the jvm settings in the `jvm.options` <<settings-files,settings file>>.
+Configure the jvm settings in the `jvm.options` <<settings-files,settings file>> or with the <<ls-java-opts, `LS_JAVA_OPTS`>>.
```
Comment (robbavey) [inline docs/static/config-details.asciidoc:4]: I'm not sure 

```suggestion
Configure JVM settings in the `jvm.options` <<settings-files,settings file>>. JVM settings can also be set via the <<ls-java-opts, `LS_JAVA_OPTS`>> environment variable
```

**Review by robbavey — CHANGES_REQUESTED** [review]
I've made some suggestions, let me know what you think

File: docs/static/config-details.asciidoc (hunk 3 above)
Comment (andsel) [reply docs/static/config-details.asciidoc:4]: Far better, my `or` wasn't an exclusive or, but this way sounds more explicit!

File: docs/static/config-details.asciidoc [hunk 4]
```diff
@@ -1,7 +1,7 @@
 [[jvm-settings]]
 === JVM settings
 
-Configure the jvm settings in the `jvm.options` <<settings-files,settings file>>.
+Configure JVM settings in the `jvm.options` <<settings-files,settings file>>. JVM settings can also be set via the <<ls-java-opts, `LS_JAVA_OPTS`>> environment variable
```
Comment (robbavey) [inline docs/static/config-details.asciidoc:4]: I missed a full stop and a line break from my original suggestion 🤦 

```suggestion
Configure JVM settings in the `jvm.options` <<settings-files,settings file>>. JVM settings can also be set via the <<ls-java-opts, `LS_JAVA_OPTS`>> environment variable.

```

File: docs/static/config-details.asciidoc [hunk 5]
```diff
@@ -111,6 +111,18 @@ java -XX:+PrintFlagsFinal -version | grep ThreadStackSize
 Depending on the default stack size, start by multiplying by 4x, then 8x, and
 then 16x until the overflow error resolves.
 
+[[ls-java-opts]]
+==== Using `LS_JAVA_OPTS`
+
+The `LS_JAVA_OPTS` environment variable can also be used to override JVM settings in the `jvm.options` file <<settings-files,settings file>>.
+The content of this variable is additive to options configured in the `jvm.options` file, and will override any settings that exist in both places
```
Comment (robbavey) [inline docs/static/config-details.asciidoc:118]: I missed a full stop from my original suggestion 🤦 

```suggestion
The content of this variable is additive to options configured in the `jvm.options` file, and will override any settings that exist in both places.
```

**Review by robbavey — COMMENTED** [review]
Couple of tiny formatting nits

**Review by robbavey — APPROVED** [review]
lgtm

**Comment by andsel** [comment]
@logstashmachine backport 8.5

**Comment by andsel** [comment]
@logstashmachine backport 7.17

**Pull request merged**